### PR TITLE
feat(esl-trigger): add `ignore-esc` attribute

### LINKED ISSUE #1110: [🚀esl-trigger]: ESC keyboard event handling
As an ESL architect, I want to have the following esl-trigger API for handling keyboard ESC events:

- Trigger send hide event if the ESC key pressed
- Trigger ignores  ESC key event if the `ignore-esc` attribute (+ property accessors) is active

### PR DESCRIPTION
**Pull request by NastaLeo**
[Closes #1110 ](https://github.com/exadel-inc/esl/issues/1110)

### CODE REVIEW

File: pages/views/examples/toggleable.njk [hunk 1]
```diff
@@ -34,7 +34,7 @@ aside:
     </div>
     <p>Triggers</p>
     <div class="btn-container">
-      <esl-trigger target="#toggleable1" mode="toggle" a11y-label-active a11y-label-inactive="expand" class="btn btn-sec-blue">Toggle</esl-trigger>
+      <esl-trigger target="#toggleable1" ignore-esc mode="toggle" a11y-label-active a11y-label-inactive="expand" class="btn btn-sec-blue">Toggle</esl-trigger>
```
Comment (ala-n) [inline pages/views/examples/toggleable.njk:37]: TODO: (as a separate ticket) I think we need a separate demo section for that

File: src/modules/esl-trigger/README.md [hunk 2]
```diff
@@ -22,6 +22,8 @@ ESLTrigger - a custom element, that allows to trigger ESLToggleable instances st
   
 - `track-hover` - [MediaQuery](../esl-media-query/README.md) to define allowed to track hover event media. (Default: `not all`)
 
+- `ignore-esc` - to disallow handle ESC keyboard event and hide target element
```
Comment (ala-n) [inline src/modules/esl-trigger/README.md:25]:
```suggestion
- `ignore-esc` - to disallow handle ESC keyboard event to hide target element
```

File: src/modules/esl-trigger/core/esl-trigger.shape.ts [hunk 3]
```diff
@@ -25,6 +25,9 @@ export interface ESLTriggerTagShape<T extends ESLTrigger = ESLTrigger> extends E
   /** Define hover event tracking media query */
   'track-hover'?: boolean | string;
 
+  /** Disallow handle ESC keyboard event and hide target element */
```
Comment (ala-n) [inline src/modules/esl-trigger/core/esl-trigger.shape.ts:28]:
```suggestion
  /** Disallow handle ESC keyboard event to hide target element */
```

File: src/modules/esl-trigger/core/esl-trigger.ts [hunk 4]
```diff
@@ -63,6 +63,9 @@ export class ESLTrigger extends ESLBaseElement {
    */
   @attr({defaultValue: '0'}) public hoverHideDelay: string;
 
+  /** Prevent ESC keyboard event handling and target element hiding */
```
Comment (ala-n) [inline src/modules/esl-trigger/core/esl-trigger.ts:66]:
```suggestion
  /** Prevent ESC keyboard event handling for target element hiding */
```

**Review by ala-n — APPROVED** [review]

**Review by abarmina — APPROVED** [review]

**Review by dshovchko — APPROVED** [review]

**Review by ala-n — APPROVED** [review]

**Pull request merged**